### DISCUSSION TITLE
Allow for separate toggling and disabling for repeated left/right commands

### DISCUSSION THREAD

**Discussion by revrev** [rxhanson/Rectangle#412] [Ideas]
<img width="412" alt="Rectangle Preferences 2021-03-17 10-41-45" src="https://user-images.githubusercontent.com/503780/111495462-62645800-870d-11eb-9b84-57dbf7173c8e.png">

I'd love the ability to disable _both_ of these behaviors, so I'm thinking separate checkboxes for each. My preferred behavior would be that repeated commands would simply do nothing. E.g. if the window is already on the right, a second move-right command would do nothing.

Thank you for all the great work building this app, I absolutely love it.

**Comment by rxhanson** [comment]
See the terminal command listed at https://github.com/rxhanson/Rectangle#halves-to-thirds-repeated-execution-of-half-and-quarter-actions

If implemented in the UI, this would probably best take the form of a dropdown, rather than an additional checkbox.

**Reply by revrev** [reply]
You're probably right, but I wonder if there may be another use case where you could have _both_ behaviors: cycle 1/2, 2/3, 1/3, move to next screen, repeat? I'm not really sure how that would work.

**Reply by rxhanson** [reply]
There actually is that behavior in there, if you read that section in the readme closely - halves to thirds for some actions & cycle displays for others. The dropdown would basically have more options. There's actually an extra option already in the app, just undocumented because I was waiting on a user to test it out (subsequentExecutionMode 4), which is move to next display on repeated execution. The unfortunate baggage that comes with UI changes is always getting out of sync with localizations, and although that doesn't deter progress, it does require a bit more motivation to make UI changes. I see no other good option than the dropdown, personally, if someone were inclined to make a contribution.

**Comment by zacharyfreiberg** [comment]
This actually helped me achieve what I was hoping to do in the discussion I started. So thanks. I do think that these should be separate settings because I was unaware of this behaviour when this setting was unticked. I just assumed it disabled the moving to different monitors. https://github.com/rxhanson/Rectangle/discussions/414
